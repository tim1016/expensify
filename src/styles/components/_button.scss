.button{
  background-color: $blue; 
  color: white;
  border: none;
  border-radius: .3rem;
  font-size: $font-size-large;
  font-weight: 300;
  padding: $s-size $m-size;
}