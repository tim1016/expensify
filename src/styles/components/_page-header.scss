.page-header{
  background: #f7f7f7;
  margin: 0 0;
  padding: $l-size 0;
  &__title{
    margin: 0;
    font-weight: 300;
    span{
      font-weight: 600;
    }
  }
}