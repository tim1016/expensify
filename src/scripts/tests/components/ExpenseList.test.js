import React from 'react';
import {ExpenseList} from '../../components/ExpenseList';
import {shallow} from 'enzyme';
import {expenses} from '../fixtures/state';

test('Render ExpenseList', ()=>{
    const wrapper = shallow(<ExpenseList expenses={expenses}/>);
    expect(wrapper).toMatchSnapshot();
})

test('Render ExpenseList (empty)', ()=>{
    const wrapper = shallow(<ExpenseList expenses={[]}/>);
    expect(wrapper).toMatchSnapshot();
})