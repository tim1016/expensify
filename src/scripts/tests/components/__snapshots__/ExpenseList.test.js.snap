// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ExpenseList (empty) 1`] = `
<div>
  <p>
    No expenses found
  </p>
</div>
`;

exports[`Render ExpenseList 1`] = `
<div>
  <ExpenseListItem
    amount={4500}
    createdAt={"1970-01-01T00:00:00.000Z"}
    description="Water bill"
    id="221e5b57-0db6-4b52-8300-fd5950e401b6"
    key="221e5b57-0db6-4b52-8300-fd5950e401b6"
    note=""
  />
  <ExpenseListItem
    amount={5000}
    createdAt={"1970-01-05T00:00:00.000Z"}
    description="Gas bill"
    id="8b80b9fe-b732-420f-934d-717ac788b94c"
    key="8b80b9fe-b732-420f-934d-717ac788b94c"
    note="Note1"
  />
  <ExpenseListItem
    amount={109500}
    createdAt={"1970-01-05T00:00:00.000Z"}
    description="Rent"
    id="7e95bfc0-900a-41da-ac66-ec442b5170dd"
    key="7e95bfc0-900a-41da-ac66-ec442b5170dd"
    note=""
  />
</div>
`;
